sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/implementation/feature/entity/FeatureFilterEvaluationContext.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.appconfiguration.config.implementation.feature.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Context passed into Feature Filters used for evaluation.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class FeatureFilterEvaluationContext {
+
+    /**
+     * Creates an instance of {@link FeatureFilterEvaluationContext}
+     */
+    public FeatureFilterEvaluationContext() {
+    }
+
+    private String name;
+
+    @JsonProperty("parameters")
+    private Map<String, Object> parameters;
+
+    private String featureName;
```

Comment:
what is the difference between `featureName` and `name`?

Reply:
This file was copied from the FM library. `featureName` isn't needed here. The `name` is the filter name, where the `featureName` is the name of the feature flag that uses it.